optimizely/client/factory_test.go

```diff
@@ -57,6 +58,26 @@ func TestClientWithSDKKey(t *testing.T) {
 	assert.NotNil(t, optimizelyClient.EventProcessor)
 }
 
+func TestClientWithPollingConfigManager(t *testing.T) {
+	factory := OptimizelyFactory{}
+
+	optimizelyClient, err := factory.Client(WithPollingConfigManager("1212", time.Second, nil))
```

Comment:
This has the side effect that it will start the polling and will be polling every second. Since we don't really care about the polling, let's set the polling interval to something really high so that we don't end up making those http requests in the test itself

Reply:
addressed